Google.Api.Generator.Utils/Formatting/WhitespaceFormatter.cs

```diff
@@ -133,15 +140,26 @@ public override SyntaxNode VisitNamespaceDeclaration(NamespaceDeclarationSyntax
             using (WithIndent())
             {
                 node = (NamespaceDeclarationSyntax)base.VisitNamespaceDeclaration(node);
+
+                // Probably the start region tag in snippets.
```

Comment:
I don't understand what this means. Are you saying we don't get here at all unless there are region tags?

Reply:
We get here every time.
I meant that if the trivia is not empty, is probably the region tag.
Before adding the region tag this was just being replaced by indents and new lines, so it was assumed that it was empty.
I've reworded the comment.